app/uk/gov/hmrc/incometaxproperty/models/errors/ServiceError.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxproperty.models.errors
+
+trait ServiceError {
```

Comment:
We also have this class somewhere, and we only use `ApiServiceError` and `DataNotFoundError`. I don't know why we have all this whole file added again. Probably a rebase issue. But we should have only the ones we use

Reply:
Actually this should not be a new change, as in main we have the class already

Reply:
Removed the unused ones.